pilot/pkg/model/authorization.go

```diff
@@ -66,7 +66,8 @@ type AuthorizationPoliciesResult struct {
 }
 
 // ListAuthorizationPolicies returns authorization policies applied to the workload in the given namespace.
-func (policy *AuthorizationPolicies) ListAuthorizationPolicies(namespace string, workload labels.Instance) AuthorizationPoliciesResult {
+// TODO: update the params in all the places this is called
+func (policy *AuthorizationPolicies) ListAuthorizationPolicies(namespace string, workloadName string, isWaypointProxy bool, workload labels.Instance) AuthorizationPoliciesResult {
```

Comment:
I don't love the extra additions to the method signature here; can you put the selection criteria in some kind of `nodeInfo`data structure?